plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -88,7 +88,7 @@ def __init__(self):
         __init__().
         """
         super(PublishBlobsStep, self).__init__(constants.PUBLISH_STEP_BLOBS,
-                                               models.Blob.TYPE_ID)
+                                               constants.BLOB_TYPE_ID)
```

Comment:
It would be good to switch both of these steps from using UnitPublishStep to use UnitModelPluginStep. The former gives you old-style AssociatedUnit instances, whereas the latter gives you new-style mongoengine ContentUnit instances. The latter also uses the very-simple query logic that we looked at in the mongoengine deep dive, whereas I think the query logic is much more complex for the old style.

It's very easy, and would look about like this as a change: https://github.com/bmbouter/pulp_rpm/pull/2/files

You might just have to access ``unit._storage_path`` instead of ``unit.storage_path``.